mmv1/products/datapipeline/product.yaml

```diff
@@ -0,0 +1,400 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: DataPipeline
+display_name: Dataflow Pipeline
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://datapipelines.googleapis.com/v1/
```

Comment:
datapipelines seems apt for the resource name. We usually go by the api endpoint.

The other resource you mentioned, `dataflow_job` uses the endpoint `https://dataflow.googleapis.com/v1b3/` which is different from what we see here: `https://datapipelines.googleapis.com/v1/`